build/pause/Makefile

```diff
@@ -12,11 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY: all push push-legacy container clean orphan
+.PHONY: all push container clean orphan all-push push-manifest
+
+include ../root/Makefile.manifest
```

Comment:
minor nit: the `build/root` directory is intended for files which live at the root of the repo, but for which we wanted to have more owners.

it seems like you just want to use this rule more generally - maybe `hack/make-rules` is a better location for this?